code/script.js

```diff
@@ -1,51 +1,285 @@
+      setTimeout (() => chooseByMood('optAngry'), 1000)
+    })
+  }
+  } 
+  
+  else if (selection === 'option2') {
+    showMessage(`Well, now that you know it works, you're more than welcome back once you need my help!`, 'bot')
+    setTimeout (() => inputWrapper.innerHTML=`
+    <p>Thanks for using this bot!</p>
+    `, 1000)
+    }
+}
+
+//Question 4: Bot suggests movie. The bot's answer depends on which Mood-button has been chosen ('optHappy', 'optSad', 'optAngry'). 
+const chooseByMood = (selection) => {
+ if (selection === 'optHappy') {
+  showMessage(`Great to hear that you are happy! Then one of these movies might suit your mood`, 'bot')
+  setTimeout (() => happyFilmButtons (), 1000)
+  
+  const happyFilmButtons = () => {
+  inputWrapper.innerHTML=`
+      <button id="happyFeet">Happy feet</button>
+      <button id="singingInTheRain">Singing in the Rain</button>
+      <button id="greaseMovie">Grease</button>
+      `
+
+      document.getElementById("happyFeet").addEventListener('click', () => {
+        showMessage(`I'll watch Happy Feet tonight!`, 'user')
+        inputWrapper.innerHTML = ''
+        setTimeout (() => lastReplyHappy('happyFeet'), 1000)
+      })
+
+      document.getElementById("singingInTheRain").addEventListener('click', () => {
+        showMessage(`I'll watch Singing in the Rain tonight!`, 'user')
+        inputWrapper.innerHTML = ''
+        setTimeout (() => lastReplyHappy('singingInTheRain'), 1000)
+      })
+
+      document.getElementById("greaseMovie").addEventListener('click', () => {
+        showMessage(`I'll watch Grease tonight`, 'user')
+        inputWrapper.innerHTML = ''
+        setTimeout (() => lastReplyHappy('grease'), 1000)
+      }) 
+    }
+  }
+
+ else if (selection === 'optSad') {
+  showMessage(`I'm so sorry to hear that you are not feeling good. Sometimes all you need is a good cry. I can help you get there.`, 'bot')
+  setTimeout (() => sadFilmButtons (), 1000)
+  
+  const sadFilmButtons = () => {
+  inputWrapper.innerHTML=`
+      <button id="titanic">Titanic</button>
+      <button id="theNotebook">The Notebook</button>
+      <button id="brokebackMountain">Brokeback Mountain</button>
+      `
+ 
+        document.getElementById("titanic").addEventListener('click', () => {
+        showMessage(`I'll watch Titanic tonight!`, 'user')
+        inputWrapper.innerHTML = ''
+        setTimeout (() => lastReplySad('titanic'), 1000)
+        })
+
+        document.getElementById("theNotebook").addEventListener('click', () => {
+        showMessage(`I'll watch The Notebook tonight!`, 'user')
+        inputWrapper.innerHTML = ''
+        setTimeout (() => lastReplySad('theNotebook'), 1000)
+        })
+
+        document.getElementById("brokebackMountain").addEventListener('click', () => {
+        showMessage(`I'll watch Brokeback Mountain tonight`, 'user')
+        inputWrapper.innerHTML = ''
+        setTimeout (() => lastReplySad('brokebackMountain'), 1000)
+        }) 
```

Comment:
Same comment as row 75. However, you are very structured and consistent with your code!